classes/controller/admin/form.ctrl.php

```diff
@@ -98,7 +99,7 @@ public function before_save($item, $data)
 
         // Registers the fields
         $fieldsConfig = \Arr::get($this->config, 'fields_config.fields');
-        foreach ($fields_data as $field_id => $field_data) {
+        foreach ($fields_data as $field_id => &$field_data) {
```

Comment:
Because the same variable name is used in the two foreach. And since the first foreach loops on the reference, not doing the same on the second one causes a weird bug (PHP replaces the content referenced by the variable instead).

Reply:
Wow that's really weird. Ok.